packages/tocco-util/src/notifier/notificationActionFactory.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {actions as toastrActions} from 'react-redux-toastr'
-import _isString from 'lodash/isString'
 import uuid from 'uuid/v4'
-import {Icon} from 'tocco-ui'
+import {Icon, IconTocco} from 'tocco-ui'
 
 import {modalComponent} from './modules/actions'
 import ModalButtons from './modules/modalComponents/ModalButtons'
 import TitleMessage from './components/TitleMessage'
 
-const NotificationIcon = icon => <Icon icon={ _isString(icon) ? icon : 'exclamation-triangle'} size="3x" />
-
 export function getInfoAction(type, title, message, icon, timeOut) {
-  const options = {
-    timeOut: timeOut,
-    showCloseButton: true,
-    component: () => <TitleMessage title={title} message={message}/>
+  const typeIconMap = {
+    error: 'times',
+    info: 'info',
+    success: 'check',
+    warning: 'exclamation'
   }
 
-  options.icon = <NotificationIcon icon={icon} />
+  type = Object.keys(typeIconMap).includes(type) ? type : 'info'
+
+  const options = {
+    attention: false,
+    component: () => <TitleMessage title={title} message={message}/>,
+    icon: <Icon icon={icon || typeIconMap[type]} size="3x" />,
+    preventDuplicates: true,
+    showCloseButton: type === 'warning' || type === 'error' || !(timeOut > 0),
```

Comment:
i think timeout <= 0 is more intuitive as ! >

Reply:
`<=` was my first choice. To handle null or undefined I have changed it to `!(timeOut > 0)`.

How can I shorten `timeOut <= 0 || timeOut === null || timeOut === undefined` with a clearer approach?

Reply:
ok haven't thought of this